app/cafe.py

```diff
@@ -0,0 +1,25 @@
+import datetime
+
+
+from app.errors import NotVaccinatedError
+from app.errors import OutdatedVaccineError
+from app.errors import NotWearingMaskError
+
+
+class Cafe:
+    def __init__(self, name: str) -> None:
+        self.name = name
+
+    def visit_cafe(self, visitor: dict) -> str:
+        if "vaccine" not in visitor:
+            raise NotVaccinatedError
```

Comment:
The checklist advises using descriptive error messages when raising exceptions. Consider providing a message for `NotVaccinatedError`, e.g., `raise NotVaccinatedError('Visitor is not vaccinated.').`